packages/control-property-editor/test/unit/panels/properties/StringEditor.test.tsx

```diff
@@ -85,6 +86,27 @@ describe('StringEditor', () => {
         fireEvent.blur(textBox);
         expect((textBox as any).value).toBe('12345');
     });
+
+    test('Do not trigger changeProperty when no change in prop value', () => {
+        const value = '1234';
+        const props: any = {
+            icons: [],
+            isEnabled: false,
+            isIcon: false,
+            name: 'testProperty',
+            value,
+            type: 'integer'
+        };
+        const spyChangeProperty = jest.spyOn(slice, 'changeProperty');
```

Comment:
Please use the `dispatch` spy like in https://github.com/SAP/open-ux-tools/blob/50515e1c75fb1e10550a0ffeb2f718a390c82910/packages/control-property-editor/test/unit/panels/properties/Funnel.test.tsx#L20-L32 instead of spying of `slice`.

Reply:
Done